opentracing-spring-cloud/src/main/java/io/opentracing/contrib/spring/cloud/jms/JmsAutoConfiguration.java

```diff
@@ -35,33 +31,7 @@
 @ConditionalOnClass({Message.class, JmsTemplate.class})
 @ConditionalOnBean(Tracer.class)
 @ConditionalOnProperty(name = "opentracing.spring.cloud.jms.enabled", havingValue = "true", matchIfMissing = true)
+@Import(TracingJmsConfiguration.class)
```

Comment:
What happend, did you move these classes to `java-jms`?

Reply:
yes, I moved them to java-jms